frontend/src/pages/draft/index.tsx

```diff
@@ -226,7 +233,7 @@ export default function Draft({ editorAPI }: { editorAPI: EditorAPI }) {
 			</div>
 		);
 	else if (generationMode === 'None' || generation === null)
-		if (!docContext.trim())
+		if (docContext.beforeCursor === '')
```

Comment:
I forget what this logic was supposed to do, but it seems like a different condition